backend/tests/lib.rs

```diff
@@ -640,6 +640,24 @@ fn test_user() {
     );
     h4.abort();
     std::thread::sleep(std::time::Duration::from_secs(1));
```

Comment:
move this to the end of this test function. In other words, insert your test code in front of it.